KsApi/models/BackingPaymentSourceTests.swift

```diff
@@ -0,0 +1,106 @@
+import Foundation
+@testable import KsApi
+import Prelude
+import XCTest
+
+final class BackingPaymentSourceTests: XCTestCase {
+  func testJSONDecoding_CompleteData() {
+    let jsonDictionary: [String: Any] = [
+      "expiration_date": "2020-02-21",
+      "id": "2",
+      "last_four": "4111",
+      "payment_type": "CREDIT_CARD",
+      "state": "ACTIVE",
+      "type": "VISA"
+    ]
+
+    let paymentSource: Backing.PaymentSource? = Backing.PaymentSource
+      .decodeJSONDictionary(jsonDictionary).value ?? nil
```

Comment:
Is it necessary to coalesce `nil` to `nil`?

Reply:
Yeah cuz `.value` actually returns an optiona `Optional<Decoded>` type 🤷‍♀